cwatm/hydrological_modules/water_quality/waterquality_main.py

```diff
@@ -102,9 +102,29 @@ def initial(self):
         
                 # only applied to landcovers with soil underneath (grassland and managed grasslands are treated has one landcover
                 for variable in phosphorusVars: vars(self.var)[variable] = np.tile(globals.inZero,(4,1))
+
             
             
             # Define depth of top soil from which nutrients leaks to runoff: 10 mm###
+
```

Comment:
This block 109-127 can be removed

Reply:
The block is already gone, since I pulled your changes.